examples/kokkos_assembly/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ if(NOT GINKGO_BUILD_EXAMPLES)
     find_package(Ginkgo 1.8.0 REQUIRED)
 endif()
 
-find_package(Kokkos REQUIRED)
+find_package(Kokkos 3.7 REQUIRED)
```

Comment:
does Kokkos cmake allow 4 is compatiable with 3.7?
Otherwise, I think it usually only allows >= 3.7 but < 4.0

Reply:
I'm not sure if I understand what you mean here.

Reply:
I thought that `find_package(Kokkos 3.7 REQUIRED)` only allows the kokkos version is >= 3.7 and < 4.0
kokkos version 4 is not allowed here, so the changes in another file are not necessary.

Reply:
No any version >= 3.7 will be accepted.

Reply:
It's a bit out of my expectation, but thanks for explanation